libs/helm/input/src/index.ts

```diff
@@ -3,8 +3,10 @@ import { HlmInput } from './lib/hlm-input';
 
 export * from './lib/hlm-input';
 
+export const HlmInputImports = [HlmInput];
```

Comment:
minor/suggestion: No strong opinions, but I think we might as well include `NgIcon` here.